server/@types/frontendComponentApi/index.d.ts

```diff
@@ -1,5 +1,9 @@
-export default interface Component {
-  html: string
-  css: string[]
-  javascript: string[]
+export interface components {
```

Comment:
Not strictly related to error handling; and what was there previously worked. I've changed it however so "structurally" it looks like a generated module/type (ie. consistent with the other modules and types that we generate from swagger specs)
The reason I made this change is a) for consistency, and also so that as and when we do import the frontend components API swagger spec (if they ever make one available), the scope of change will be minimal 👍